test/common.bash

```diff
@@ -133,75 +133,75 @@ chmod 700 "$btnew"
 export BATS_TMPDIR=$btnew
 [[ $(stat -c %a "$BATS_TMPDIR") = '700' ]]
 
-# Separate directories for tarballs and images
-TARDIR=$CH_TEST_TARDIR
-IMGDIR=$CH_TEST_IMGDIR
-
 # MPICH requires different handling from OpenMPI. Set a variable to enable
 # some kludges.
 if [[ $BATS_TEST_DIRNAME =~ 'mpich' ]]; then
-    CHTEST_MPI=mpich
+    ch_mpi=mpich
     # First kludge. MPICH's internal launcher is called "Hydra". If Hydra sees
     # Slurm environment variables, it tries to launch even local ranks with
     # "srun". This of course fails within the container. You can't turn it off
     # by building with --without-slurm like OpenMPI, so we fall back to this
     # environment variable at run time.
     export HYDRA_LAUNCHER=fork
 else
-    CHTEST_MPI=openmpi
+    ch_mpi=openmpi
 fi
 
 # Crays are special.
 if [[ -f /etc/opt/cray/release/cle-release ]]; then
-    CHTEST_CRAY=yes
+    ch_cray=yes
 else
-    CHTEST_CRAY=
+    ch_cray=
 fi
 
+# Separate directories for tarballs and images
+ch_imgdir=$CH_TEST_IMGDIR
+ch_tardir=$CH_TEST_TARDIR
+
 # Some test variables
-EXAMPLE_TAG=$(basename "$BATS_TEST_DIRNAME")
-EXAMPLE_IMG=$IMGDIR/$EXAMPLE_TAG
-CHTEST_TARBALL=$TARDIR/chtest.tar.gz
-CHTEST_IMG=$IMGDIR/chtest
```

Comment:
>Because this one is used so pervasively, I think we should consider calling it img without the prefix, as a special case. I'd like to hear your opinion, though. If we agree on that, you could add a comment to that effect.

I am reluctant to use `img` as the special case variable for `ch_chtest_img` because `img` appears to be used as a local variable for other scripts, e.g., `test/chtest/Build`. Although that particular script doesn't load `common.bash` and thus there are no conflicts, I would prefer to keep the `ch_chtest_img` variable unique. Perhaps we should use a special prefix? Something like `cht_img`, as you had suggested in the past?